### ISSUE TITLE
Extension is not loading

### ISSUE THREAD

**Issue by renannprado** [material-shell/material-shell#220]
**Describe the bug**
Extension is not loading

**Expected behavior**
Extension should load and work.

**Screenshots**

![image](https://user-images.githubusercontent.com/2359543/87942207-88612c00-ca9c-11ea-9ff7-072d27a9ae56.png)

**To Reproduce**
- Occurs on:
  - [ ] X.Org
  - [x] Wayland
- GNOME Shell version: 3.30.2
- Distro: Debian GNU/Linux 10 (buster)

**Current commit**
Output from `git rev-parse --short HEAD`: `b789279` (v1).

I also tried `d4575b4` (2-beta) and `d4575b4` (master).

**Log data**
Output from `journalctl /usr/bin/gnome-shell`:

Latest version I tried `b789279` is having this error:
```
Jul 20 15:08:28 computer gnome-shell[11314]: JS WARNING: [/home/user/.local/share/gnome-shell/extensions/material-shell@papyelgringo/module/disableIncompatibleExtensionsModule.js 47]: reference to undefined prop
Jul 20 15:08:28 computer gnome-shell[11314]: JS ERROR: TypeError: Main.extensionManager is undefined
                                            enable@/home/user/.local/share/gnome-shell/extensions/material-shell@papyelgringo/module/disableIncompatibleExtensionsModule.js:47:1
                                            enable/<@/home/user/.local/share/gnome-shell/extensions/material-shell@papyelgringo/extension.js:49:9
```

The other two versions I tried I was facing:

```
Extension "material-shell@papyelgringo" had error: Error: No signal 'closing' on object 'MetaDisplay'
```

**Comment by PapyElGringo** [comment]
Hey as you said your gnome version is :
> * GNOME Shell version: 3.30.2

And as you can see in the readme.md we require a higher version of Gnome Shell

> REQUIRES: gnome-shell >= 3.34.0

You need to update your system somehow !

**Comment by renannprado** [comment]
@PapyElGringo is there any earlier version that supports gnome shell 3.30.x? otherwise I'll see if I can update it. I'm a little bit afraid 'cause this newer version is not part of the latest updates Debian is offering me and don't want to break the whole thing by doing it.

**Comment by PapyElGringo** [comment]
Well in 3.30 Material Shell didn't existed yet ! Debian is very late in Gnome updates

**Comment by renannprado** [comment]
@PapyElGringo that sucks, I was excited to use this extension. I hope soon I'll have time to install Manjaro and there I suppose I wouldn't face the same issue. Thanks anyway!
